app/src/components/blocks/selectStage/SelectStage.test.tsx

```diff
@@ -58,16 +65,14 @@ describe('<UploadDocumentsPage />', () => {
             });
 
             act(() => {
-                userEvent.upload(screen.getByTestId('LG-input'), [
-                    documentOne,
-                    documentTwo,
-                    documentThree,
-                ]);
+                userEvent.upload(screen.getByTestId('LG-input'), [lgDocumentOne, lgDocumentTwo]);
```

Comment:
Does eslint not throw an error when there is no assertion inside an act statement?

Reply:
No, didn't get an eslint error that.